admiral/pkg/clusters/dynamoDB.go

```diff
@@ -205,6 +219,52 @@ func (client *DynamoClient) getWorkloadDataItemByIdentityAndEnv(env, identity, t
 	return workloadDataItems, nil
 }
 
+func (client *DynamoClient) getDynamicConfig(key string, value string, tableName string) (DynamicConfigData, error) {
+
+	configData := DynamicConfigData{}
+
+	keyCond := expression.KeyEqual(expression.Key(key), expression.Value(value))
+
+	expr, err := expression.NewBuilder().
+		WithKeyCondition(keyCond).
+		Build()
+
+	if err != nil {
+		return configData, err
+	}
+
+	dbQuery := dynamodb.QueryInput{
+		TableName:                 aws.String(tableName),
+		ExpressionAttributeNames:  expr.Names(),
+		ExpressionAttributeValues: expr.Values(),
+		KeyConditionExpression:    expr.KeyCondition(),
+		FilterExpression:          expr.Filter(),
+	}
+
+	items, err := client.svc.Query(&dbQuery)
+
+	if err != nil {
+		return configData, fmt.Errorf("Failed to fetch dynamic config : %s", err)
```

Comment:
can you make sure we have this is a format where we can easily search example task=DynamicConfigGet etc in the log